### DISCUSSION TITLE
Автоустановка на Ubuntu 20 с доменом и SSL-сертификатом!

### DISCUSSION THREAD

**Discussion by DanilfromRussia** [totumonline/totum-ru-issues-and-discussions#444] [Как это сделать?] [answered]
Добрый день, мой домен второго уровня (abcde.ru) куплен и делегирован на IP офиса в другом городе, totum-mit пытаюсь поставить на домен третьего уровня (example.abcde.ru) и дилегировать на IP виртуалки в локальной сети. Cмог миновать ошибку проверки LetsEncrypt http-01 переписав скрипт autoinstall.sh на проверку dns-01, однако все равно скрипт в процессе установки выдает ошибки неудачной проверки http-01 и как итог "Не удается получить доступ к сайту" при переходе на url

Вопрос: Связана ли проблема с "невалидным" поддоменом и нужно ли после установки запускать docker-compose (т.к после autoinstall у меня не создан ни один из docker-контейнеров и не скачан ни один из docker images)

UPD: Хрен пойми как он создался без единого docker контейнера и без регистрации в nginx proxy manager (хотя я накатывал его больше 5 раз подряд игнорируя ошибки, что противоречит правилу установки на чистую виртуалку), в общем проблема решена

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Мы вам врядли чем-то поможем. Единственно что не рекомедуем docker-версию тк она давно нами не поддерживается. Там есть ссылка на какой-то репо пользователя, который делал правильный компоузе, но мы к нему отношения не имеем. Мы обслуживаем только скрипты установок в основном репо. Со всеми вариантами пробросов - это не к нам.
